pymc3/stats.py

```diff
@@ -342,7 +342,7 @@ def bpic(trace, model=None):
 
 
 def compare(traces, models, ic='WAIC', method='stacking', b_samples=1000,
-            alpha=1, seed=None):
+            alpha=1, seed=None, round_to=2):
```

Comment:
I agree "2" is arbitrary. Nevertheless, I think is _not that bad_ as a default value even with the risk of eventually masking differences, and thus to be forced to re-run compare with a larger number of decimals.

Anyway, maybe for consistency, we could remove the rounding. If, in the future we decide to change the _de facto no-rounding policy_, we can revisit this. What do you think?